.dotnet/src/Custom/FineTuning/FineTuningClient.cs

```diff
@@ -62,4 +64,50 @@ protected internal FineTuningClient(ClientPipeline pipeline, Uri endpoint, OpenA
+    /// <param name="training_file"> The training file name that is already uploaded. String should match pattern '^file-[a-zA-Z0-9]{24}$'  </param>
+    /// <param name="model"> The model name to fine-tune. String such as "gpt-3.5-turbo" </param>
+    /// <param name="options"> Additional options (<see cref="RequestOptions"/>) to customize the request. </param>
+    public ClientResult<FineTuningJob> CreateJob(string training_file, string model, RequestOptions options = default)
+    {
+        var request = new InternalCreateFineTuningJobRequest(model, training_file);
+        var content = request.ToBinaryContent();
+        ClientResult result = CreateJob(content, options);
+        return ClientResult.FromValue(FineTuningJob.FromResponse(result.GetRawResponse()), result.GetRawResponse());
+    }
+
+    /// <summary> Async version of create job</summary>
+    /// <param name="training_file"> The training file name that is already uploaded. String should match pattern '^file-[a-zA-Z0-9]{24}$'  </param>
+    /// <param name="model"> The model name to fine-tune. String such as "gpt-3.5-turbo" </param>
+    /// <param name="options"> Additional options (<see cref="RequestOptions"/>) to customize the request. </param>
+    public async Task<ClientResult<FineTuningJob>> CreateJobAsync(string training_file, string model, RequestOptions options = default)
+    {
+        var request = new InternalCreateFineTuningJobRequest(model, training_file);
+        var content = request.ToBinaryContent();
+        ClientResult result = await CreateJobAsync(content, options);
+        return ClientResult.FromValue(FineTuningJob.FromResponse(result.GetRawResponse()), result.GetRawResponse());
+    }
+
+    /// <summary>
+    /// Cancels a fine-tuning job with the specified job ID.
+    /// </summary>
+    /// <param name="jobId">The ID of the job to cancel.</param>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns>A <see cref="ClientResult{FineTuningJob}"/> containing the canceled fine-tuning job.</returns>
+    public ClientResult<FineTuningJob> CancelJob(string jobId, CancellationToken cancellationToken = default)
+    {
+        ClientResult result = CancelJob(jobId, cancellationToken.ToRequestOptions());
+        return ClientResult.FromValue(FineTuningJob.FromResponse(result.GetRawResponse()), result.GetRawResponse());
+    }
+
+    /// <summary> Async version of cancel job</summary>
+    /// <param name="jobId"> The job ID to cancel. </param>
+    /// <param name="cancellationToken"> The cancellation token. </param>
+    /// <returns> A <see cref="Task{T}"/> of <see cref="ClientResult{FineTuningJob}"/> containing the canceled fine-tuning job. </returns>
+    public async Task<ClientResult<FineTuningJob>> CancelJobAsync(string jobId, CancellationToken cancellationToken = default)
+    {
+        ClientResult result = await CancelJobAsync(jobId, cancellationToken.ToRequestOptions());

```

Comment:
Generally, awaited async method calls inside of libraries always need `.ConfigureAwait(false)` at the end -- separate topic for .NET guts, but it has to do with ensuring thread continuation contexts do the correct/expected things when you're consuming the library (especially if there's any UI interaction involved)